waspc/data/Generator/templates/react-app/src/auth/forms/SocialIcons.jsx

```diff
@@ -0,0 +1,34 @@
+import { css } from '@stitches/react'
```

Comment:
We should write all of our internal components with Typescript IMHO, I would suggest converting all `jsx` files to `tsx` and then typing all the props the components receive.

Reply:
agreed! Will do